rebound/simulation.py

```diff
@@ -413,7 +413,8 @@ def initSimulationArchive(self, filename, interval=None, interval_walltime=None)
         interval_walltime : float
             Interval between outputs in wall time (seconds). Useful for adaptive timesteps. 
         """
-        self.ri_whfast.safe_mode=0
+        if cast(self._post_timestep_modifications, c_void_p).value is not None:
```

Comment:
Why does it not work with PTM? Sorry if we already talked about it and I forgot.